### DISCUSSION TITLE
Private League Recent Activity Not Returning Anything

### DISCUSSION THREAD

**Discussion by tkauffman42** [cwendt94/espn-api#323] [ESPN API] [answered]
First wanted to say I love the project and the help you have offered with this. I have been having a problem though with a private league getting the recent activity.  I have the year, swid and ESPN_s2 as variables.  I can also get everything else using the API, so I know that those values are correct.

`league = League(league_id, year, espn_s2=espn_s2, swid=swid)`
`activity = league.recent_activity()`
`print(len(activity))`

When I do that it always returns 0.  I have tried adding a size and even a msg_type and still nothing different.  I have gone to ESPN's fantasy page and there is recent activity (I dropped some players for testing purposes).

Any help or direction would be appreciated.

**Comment by dtcarls** [comment]
What sport are you using it on? Has the `year` provided as an argument of that league started? I am successful in getting recent activity data of a football private league with year 2021.

**Reply by tkauffman42** [reply]
It was Football.  I was using 2021 as my year, as the 2022 year returned an error (and I double checked ESPN and they also showed of having the year be 2021)
![recentactivity](https://user-images.githubusercontent.com/83241365/159273276-8429c0b6-5e94-4526-9248-6e320fab79af.png)

**Reply by dtcarls** [reply]
Apologies, not sure how to help if I cannot reproduce the issue. Maybe someone else on the project can reproduce and provide some insight.

**Reply by tkauffman42** [reply]
Are you the admin of your league and do you think that has anything to do with it?

**Comment by tkauffman42** [comment, marked as answer]
Found the error on my side.

**Reply by cwendt94** [reply]
Glad you were able to find the issue!
